### ISSUE TITLE
How to communicate level of expertise on offered services?

### ISSUE THREAD

**Issue by akapug** [simbi-inc/planning#10]

**Comment by akapug** [comment]
One end of the options spectrum would involve asking people to self-certify expertise level on Services.

The other end of the spectrum would involve algorithmically assigning expertise at the category level. 

Lots of nuance to assess here!

**Comment by Carzwell** [comment]
Yes, sir.  I have the idea of self-vetting and community-vetting.  One may declare whatever they want.  The proof is in the pudding, and service excellence.  

There is this, too: I could offer a service as a MSSQL Server Administrator, and set myself at the highest level.  That would be appropriate given my experience and knowledge.  Now, if I try putting a Wordpress service up there?  I'm an OK novice, you know what I mean?  I'm not advanced, by any means.

But, as a novice with a lower posted rate, I can be available to help those with smaller budgets and easier requirements.  If I posted myself on Simbi as a DBA, I would have to break the bank.  :-)
